packages/aws-rfdk/bin/stage-deadline.ts

```diff
@@ -12,7 +12,12 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as url from 'url';
 import { types } from 'util';
-import {IUris, Platform, Product, VersionProvider } from '../lib/core/lambdas/nodejs/version-provider';
+import {
+  IUris,
+  Platform,
+  Product,
+  VersionProvider,
+} from '../lib/core/lambdas/nodejs/version-provider/version-provider';
```

Comment:
This fix, to me, says that `version-provider.ts` should actually have been in a `lib/core/lambdas/nodejs/lib/version-provider/` rather than in the same module as the `VersionQuery` custom resource handler.

Reply:
I just approved this PR, but this suggestion makes sense to me as well.

Reply:
Do you think we should be doing this now given `stage-deadline` doesn't work for RFDK 0.18.0? Making that kind of change would probably warrant more testing and potentially cause conflicts in #175. How would you feel about postponing that change to be in #175? Or possibly as a part of #146?

Reply:
#175 is in-progress and can be rebased easily enough. The only live code that uses `version-provider.ts` is this stage-deadline script. IMO, I'd rather just do the correct fix and force an update of #175 rather than go with a quick-fix now and kick the can down the road.

Reply:
I've made the change.